.gitignore

```diff
@@ -0,0 +1,2 @@
+*.css
+*.wxss
```

Comment:
Where does this file extension come from?

Reply:
From my IDE-Plugin, I can move it to my local gitignore if you prefer.